cnf-certification-test/suite_test.go

```diff
@@ -56,8 +62,10 @@ const (
 	defaultClaimPath              = ".."
 	defaultCliArgValue            = ""
 	junitFlagKey                  = "junit"
+	serverRun                     = "runserver"
```

Comment:
add support for --runserver  in run-cnf-suites.sh script

Reply:
i did that and it removed by mistake